src/app/routes/getInitialData/article/index.test.js

```diff
@@ -1,133 +1,38 @@
-import baseUrl from '../utils/getBaseUrl';
-import onClient from '#lib/utilities/onClient';
 import fetchData from '../utils/fetchData';
+import applyTimestampRules from '#lib/utilities/preprocessor/rules/timestamp';
+import addIdsToBlocks from '#lib/utilities/preprocessor/rules/addIdsToBlocks';
+import applyBlockPositioning from '#lib/utilities/preprocessor/rules/blockPositioning';
 
-const mockApplyTimestampRules = jest.fn();
```

Comment:
any reason the preprocessor rule tests have been removed?

Reply:
The test is still there only we don't need to mock `preprocessorRules` anymore. https://github.com/bbc/simorgh/pull/4185/files/57cbf21f659c953551bcb27e92f0f655fa8368df#diff-2380915dc9f3747c991fc8574fb6bebeR8